src/test/mochitest/head.js

```diff
@@ -223,6 +224,10 @@ function waitForSelectedSource(dbg, sourceId) {
   });
 }
 
+function isSourceLoaded(source) {
```

Comment:
we have this defined in source-tree/utils -- could we use that, or is this playing a special role?